e2e_tests/tests/fixtures/generic_task/test_config_fork.yaml

```diff
@@ -0,0 +1 @@
+entrypoint: ["python3", "run_fork.py"]
```

Comment:
why not delete `run_fork.py` and change this to `entrypoint: ["echo", "forked"]`?